burst/README.md

```diff
@@ -0,0 +1,87 @@
+# Burst Metrics Service
+
+Burst metrics service is responsible for reading HPA metrics from a Kubernetes(k8s) API and return a formatted burst metrics header. It is implemented as a REST service.
+
+Currently it will capture current pod count and target pod count from an HPA.
+
+## Deployment/usage
+
+TODO: add recently pushed `make` usage
+
+## API Endpoints
+
+### `/burstmetrics/{namespace}/{HPA}`
+
+This is the only endpoint exposed by this service.
+This endpoint will check the k8s `{namespace}` for the specified `{HPA}`.
+
+If the `{HPA}` is found and is accessible, it will return the metrics below in a formatted fashion:
+
+- service=namespace/HPA-name
+- current-load=current pod count
+- max-load=target hpa pod count
+- target-load=80% of max-load <!-- WIP:Configure the percentage -->
```

Comment:
I would remove the WIP until it's implemented to avoid confusion

Reply:
Updated